integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/nextbest/NextBestQueue.java

```diff
@@ -46,7 +46,13 @@ public void enqueueEduDocument(EduDocument eduDocument) throws MessageException
         }
 
         byte[] decryptedAsicPackage = decrypt((Payload)eduDocument.getAny());
-        List<String> contentFromAsic = getContentFromAsic(decryptedAsicPackage);
+        List<String> contentFromAsic = null;
+        try {
+            contentFromAsic = getContentFromAsic(decryptedAsicPackage);
+        } catch (MessageException e) {
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either log or rethrow this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1166)